easy-ui-tokens/src/shape/base.json

```diff
@@ -0,0 +1,16 @@
+{
+  "shape": {
+    "border_radius": {
```

Comment:
renamed this to border_radius to be parallel with border_width and since that's what it is, but per conversation in Slack there might be a better way to organize these in general at some point